src/components/inputs/ImageInput.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { View, Text } from 'react-native'
+import cn from 'classnames'
+import { ImageInputSize } from '@/types/imageInput'
+import Icon from '../Icon'
```

Comment:
@/components/Icon

Reply:
Auto-import keeps doing that :(

Fixed now :)